gradle.properties

```diff
@@ -1,6 +1,4 @@
-## Project-wide Gradle settings.
```

Comment:
Are any of the changes to this file important for this pull request? If not, would you mind removing this file from this pull request?

If you really want to improve this file, you are free to open a separate pull request for it though. :-)

Reply:
I have removed this file from my commit. It got modified automatically due to my Android studio settings.